frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -346,6 +344,9 @@ def include_attachments(self, message):
 
 		return safe_encode(message_obj.as_string())
 
+	def get_email_retry_limit():
```

Comment:
this can just be a simple function/staticmethod? 🤔 - seems like a util

Also, this is being used as a (class) method, but there is no `self` param present